src/main/java/com/epam/bookstoreservice/dto/response/Result.java

```diff
@@ -8,29 +8,29 @@
 @Setter
 public class Result<T> {
```

Comment:
Try to remove this entity.
Use ResponseEntity.status(HttpStatus.XXX).body(xxx) instead